src/components/pages/TicketsDashboard/TicketsDashboard.js

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect, useState } from 'react';
+import axiosWithAuth from '../../../utils/axiosWithAuth';
+import { Statistic, Row, Col, Table } from 'antd';
+// TODO: update page styling and functionality, see wireframes/NewDesignProposition/AdminWireframes/Dashboard
+
+const columns = [
+  {
+    title: 'Ticket ID',
+    dataIndex: 'ticketID',
+    filters: [
+      {
+        text: 'Example',
+        value: 'Example',
+      },
+      {
+        text: 'Jim',
+        value: 'Jim',
+      },
+      {
+        text: 'Submenu',
+        value: 'Submenu',
+        children: [
+          {
+            text: 'Green',
+            value: 'Green',
+          },
+          {
+            text: 'Black',
+            value: 'Black',
+          },
+        ],
+      },
+    ],
+    // specify the condition of filtering result
+    // here is that finding the name started with `value`
+    onFilter: (value, record) => record.ticketID.indexOf(value) === 0,
+    sorter: (a, b) => a.ticketID - b.ticketID,
+    sortDirections: ['descend'],
+  },
+  {
+    title: 'Message',
+    dataIndex: 'message',
+    filters: [
+      {
+        text: 'London',
+        value: 'London',
+      },
+      {
+        text: 'New York',
+        value: 'New York',
+      },
+    ],
+
+    onFilter: (value, record) => record.ticketType.indexOf(value) === 0,
+  },
+  {
+    title: 'Date Submitted',
+    dataIndex: 'dateSubmitted',
+    defaultSortOrder: 'descend',
+    sorter: (a, b) => a.dateSubmitted < b.dateSubmitted,
+  },
+];
+
+function onChange(pagination, filters, sorter, extra) {
+  // console.log('params', pagination, filters, sorter, extra);
+}
+
+// TODO: make Ant Design Statistics pull ticket totals from ticket tables
+const TicketsDashboard = props => {
+  const [tickets, setTickets] = useState([]);
+
+  useEffect(() => {
+    const getTickets = () => {
+      axiosWithAuth()
+        .get('/resource-tickets')
+        .then(res => {
+          if (res.data.message === null) {
+            setTickets(res.data);
+          }
+        });
+    };
+    getTickets();
+  }, []);
+
+  const data = [];
+  let escaTickets = [];
+  if (tickets !== []) {
+    escaTickets = tickets.filter(x => x.ticket_status === 'approved');
+  }
+  // console.log('escaTickets', escaTickets);
+  // eslint-disable-next-line array-callback-return
```

Comment:
Please don't disable the linter. conform to it's standards :)

Reply:
Fixed it on last commit